whatshap/cli/split.py

```diff
@@ -48,12 +50,15 @@ def add_arguments(parser):
         'and the haplotag list file.')
     arg('--read-lengths-histogram', default=None,
         help='Output file to write read lengths histogram to in tab separated format.')
+    arg('--split-by-cluster', default=False, action='store_true',
+        help='Split by given (chromosome) clusters.')
     arg('reads_file', metavar='READS', help='Input FASTQ/BAM file with reads (FASTQ can be gzipped)')
     arg('list_file', metavar='LIST',
-        help='Tab-separated list with (at least) two columns <readname> and <haplotype> (can be gzipped). '
-        'Currently, the two haplotypes have to be named H1 and H2 (or none). Alternatively, the '
-        'output of the "haplotag" command can be used (4 columns), and this is required for using '
-        'the "--only-largest-block" option (need phaseset and chromosome info).')
+        help='Tab-separated list with (at least) two columns <readname> and <haplotype>, or at least '
+            'three columns <readname>, <chromosome-direction cluster>, and <chromosome cluster> (can be gzipped). '
```

Comment:
I assume you think about "Watson/Crick" directionality here, but that's more guessing, i.e. I think the name "chromosome-direction cluster" won't be understood by many users. Could this be renamed - or described - to make clear what the meaning is?

Reply:
Yes that was what I meant. This is the more exact explanations with the input requirements. Maybe we can add these lines? 
'chromosome/direction clusters indicate the original chromosome and forward/backward direction of reads.'
'each of these cluster columns should be a set of integers in the range [1:n], where n is the number of clusters.' 
'For example in female human genome, the range of second and third columns are [1:46] and [1:23], respectively.'

Reply:
That sounds much better, yet I think we need to clarify that Strand-seq is used to generate this input (otherwise, I fear that there will be many support tickets asking how to generate the input using Illumina WGS)